### PR TITLE
Add open-vm-tools and virtualbox-guest-utils

### PR DESCRIPTION
**Pull request by axtloss**
this gives a better experience when using crystal with vmware or virtualbox

### CODE REVIEW

File: chrooted-iso.sh [hunk 1]
```diff
@@ -56,4 +52,8 @@ echo "en_US.UTF-8 UTF-8" >> /etc/locale.gen
 echo "LANG=en_US.UTF-8" > /etc/locale.conf
 locale-gen
 
+# Onyx as default gdm session
```
Comment (SomethingGeneric) [inline chrooted-iso.sh]: As long as this is tested im fine with all these changes

**Review by SomethingGeneric — COMMENTED** [review]
._.

**Comment by mchalunderscore** [comment]
@axtloss Why does this remove my *functional* AccountsService code?

**Comment by axtloss** [comment]
that is an excellent question, the answer to that question is: I have no idea

**Comment by axtloss** [comment]
Maybe the development branch didn't have these changes? Since that's what i based the new branch on

**Comment by mchalunderscore** [comment]
:gun: Fix it

**Comment by axtloss** [comment]
fixed it

File: chrooted-iso.sh (hunk 1 above)
Comment (SomethingGeneric) [reply chrooted-iso.sh]: @axtloss ?

Comment (axtloss) [reply chrooted-iso.sh]: Yeah this is tested, I thought you were saying something about the onyx gdm thing

**Comment by SomethingGeneric** [comment]
@not-my-segfault good now?

**Review by SomethingGeneric — APPROVED** [review]
LFG it LGTM

**Pull request merged**